### PR TITLE
Added a getting started guide & updated the user landing page flow

### PR DESCRIPTION
**Pull request by Millu**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [X] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [ ] Yes
- [X] No, because: Just a documentation update

      
## Have you updated all relevant documentation?
- [X] Yes
- [ ] No

## Description
Updated documentation with a getting started guide & a glossary of terms needed to get started
Updated the landing page flow for users 

Screenshot 2023-07-27 at 9 53 25 PM

## Related Tickets & Documents

- Related Issue #
- Closes #

## Added/updated tests?

- [ ] Yes
- [ ] No : _please replace this line with details on why tests
      have not been included_

### CODE REVIEW

File: docs/help/gettingStartedWithAI.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+### Generation Workflows
+
+- Invoke offers a number of different workflows for interacting with models to produce images. Each is extremely powerful on its own, but together provide you an unparalleled way of producing high quality creative outputs that align with your vision.
+    - **Text to Image:** The text to image tab focuses on the key workflow of using a prompt to generate a new image. It includes other features that help control the generation process as well.
+    - **Image to Image:** With image to image, you provide an image as a reference (called the “initial image”), which provides more guidance around color and structure to the AI as it generates a new image. This is provided alongside the same features as Text to Image.
+    - **Unified Canvas:** The Unified Canvas is an advanced AI-first image editing tool that is easy to use, but hard to master. Drag an image onto the canvas from your gallery in order to regenerate certain elements, edit content or colors (known as inpainting), or extend the image with an exceptional degree of consistency and clarity (called outpainting).
+
+### Improving Image Quality
```
Comment (hipsterusername) [inline docs/help/gettingStartedWithAI.md:34]: We may need to update for sdxl

File: docs/help/gettingStartedWithAI.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- Increasing Steps - The number of steps used controls how much time the model is given to produce an image, and depends on the “Scheduler” used. The schedule controls how each step is processed by the model. More steps tends to mean better results, but will take longer - We recommend at least 30 steps for most
+- Tweak and Iterate - Remember, it’s best to change one thing at a time so you know what is working and what isn't. Sometimes you just need to try a new image, and other times using a new prompt might be the ticket. For testing, consider turning off the “random” Seed - Using the same seed with the same settings will produce the same image, which makes it the perfect way to learn exactly what your changes are doing.
+- Explore Advanced Settings - InvokeAI has a full suite of tools available to allow you complete control over your image creation process - Check out our [docs if you want to learn more](https://invoke-ai.github.io/InvokeAI/features/).
+
+## Terms & Concepts
```
Comment (hipsterusername) [inline docs/help/gettingStartedWithAI.md:43]: May need addition of Sdxl terms

File: docs/installation/INSTALLATION.md [hunk 3]
```diff
@@ -15,6 +13,48 @@ See the [troubleshooting
...
+  - [CUDA and ROCm Drivers](installation/030_INSTALL_CUDA_AND_ROCM.md)
+  - [Installing New Models](installation/050_INSTALLING_MODELS.md)
+
+## :fontawesome-solid-computer: Hardware Requirements
+
+### :octicons-cpu-24: System
+
+You wil need one of the following:
+
+- :simple-nvidia: An NVIDIA-based graphics card with 4 GB or more VRAM memory.
```
Comment (hipsterusername) [inline docs/installation/INSTALLATION.md:39]: Will probably need to call out SDXL specs

File: docs/help/gettingStartedWithAI.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+Embeddings, like LoRAs, assist with more easily prompting for certain characters, styles, or concepts. However, embeddings are trained to more update the relationship between a specific word (known as the “trigger”) and the intended output. Embeddings may sometimes also be referred to as Textual Inversions (TIs).
+
+### ControlNet
+
+ControlNet is a neural network model that can be used to control output from models. This can take many forms, such as controlling poses of people in generated images or providing edges to based image generation on. The impact of the ControlNet can also be adjusted to increase or decrease the similarity of the generated image to the ControlNet.
+
+### VAE
+
+Variational auto-encoder (VAE) is a generative AI algorithm that helps to generate finer details such as better faces, hands, colors etc.
```
Comment (lstein) [inline docs/help/gettingStartedWithAI.md:93]: This is not technically correct. The VAE is an encode/decode model that translates between the small "latents" image that the image generation process produces, and the large pixel images that we see. Some VAEs do that job better than others, and updating to a more recent VAE often leads to improved image and color fidelity.

File: docs/help/gettingStartedWithAI.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+Low-Rank Adaptations (LoRAs) ****are like a smaller, more focused version of model, intended to focus on training a better understanding of how a specific character, style, or concept looks.
+
+### Embeddings
+
+Embeddings, like LoRAs, assist with more easily prompting for certain characters, styles, or concepts. However, embeddings are trained to more update the relationship between a specific word (known as the “trigger”) and the intended output. Embeddings may sometimes also be referred to as Textual Inversions (TIs).
+
+### ControlNet
+
+ControlNet is a neural network model that can be used to control output from models. This can take many forms, such as controlling poses of people in generated images or providing edges to based image generation on. The impact of the ControlNet can also be adjusted to increase or decrease the similarity of the generated image to the ControlNet.
```
Comment (lstein) [inline docs/help/gettingStartedWithAI.md:89]: I would change to "ControlNets are neural network models that are able to extract key features from an existing image and use these features to guide the output of the image generation model."

File: docs/help/gettingStartedWithAI.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+Schedulers can be intricate and there's often a balance to strike between how quickly they can de-noise data and how well they can do it. It's typically advised to experiment with different schedulers to see which one gives the best results. There has been a lot written on the internet about different schedulers, as well as exploring what the right level of "steps" are for each. You can save generation time by reducing the number of steps used, but you'll want to make sure that you are satisfied with the quality of images produced!
+
+### Low-Rank Adaptations / LoRAs
+
+Low-Rank Adaptations (LoRAs) ****are like a smaller, more focused version of model, intended to focus on training a better understanding of how a specific character, style, or concept looks.
+
+### Embeddings
+
+Embeddings, like LoRAs, assist with more easily prompting for certain characters, styles, or concepts. However, embeddings are trained to more update the relationship between a specific word (known as the “trigger”) and the intended output. Embeddings may sometimes also be referred to as Textual Inversions (TIs).
```
Comment (lstein) [inline docs/help/gettingStartedWithAI.md:85]: These should be called "Textual Inversion Embeddings." `Embedding` is just a generic term for weights that can be attached to a portion of a pre-trained model.

File: docs/help/gettingStartedWithAI.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+- Increasing Steps - The number of steps used controls how much time the model is given to produce an image, and depends on the “Scheduler” used. The schedule controls how each step is processed by the model. More steps tends to mean better results, but will take longer - We recommend at least 30 steps for most
+- Tweak and Iterate - Remember, it’s best to change one thing at a time so you know what is working and what isn't. Sometimes you just need to try a new image, and other times using a new prompt might be the ticket. For testing, consider turning off the “random” Seed - Using the same seed with the same settings will produce the same image, which makes it the perfect way to learn exactly what your changes are doing.
+- Explore Advanced Settings - InvokeAI has a full suite of tools available to allow you complete control over your image creation process - Check out our [docs if you want to learn more](https://invoke-ai.github.io/InvokeAI/features/).
+
+## Terms & Concepts
+
+### Stable Diffusion
```
Comment (lstein) [inline docs/help/gettingStartedWithAI.md:45]: You are welcome to link to a slide deck I created from my laboratory which describes the Stable Diffusion algorithm in layman's terms: https://docs.google.com/presentation/d/1IO78i8oEXFTZ5peuHHYkVF-Y3e2M6iM5tCnc-YBfcCM/edit?usp=sharing

**Review by lstein — CHANGES_REQUESTED** [review]
I'm partially done with the review. This is a checkpoint.

File: docs/help/gettingStartedWithAI.md (hunk 2 above)
Comment (Millu) [reply docs/help/gettingStartedWithAI.md:43]: Any others that should be included outside of style and refiner?

**Review by lstein — APPROVED** [review]
My comments have been addressed. Thanks!
